html-css/modifier-utilities/step4-instructions.md

```diff
@@ -17,37 +21,59 @@ Click the <strong>Copy to Editor</strong> button below to add html for a Label t
 &lt;/span>
 </pre>
 
-2) <strong>Copy more code to the editor.</strong>
+2) <strong>Set up the styles file in `myapp.css`.</strong>
 
-Click the <strong>Copy to Editor</strong> button below to add the template for your new styles.
+Click the `Copy to Editor` button below to add the template for the new styles.
 
 <pre class="file" data-filename="myapp.scss" data-target="replace">
 .pf-c-label {
   // Styles go here
 }
 </pre>
 
-3) In `myapp.scss` add a new block inside of the `.pf-c-label` class that will add styles for the warning modifier:
+3) <strong>Decide on which styles to override for the component.</strong>
+
+When adding new style overrides for a component decide on which properties to change. For example, in the case of the label, overriding the background color is a suitable choice because the component already styles the background color.
```

Comment:
```suggestion
When adding new style overrides for a component, decide on which properties to change. For example, in the case of the label, overriding the background color is a suitable choice because the component already styles the background color.
```